### ISSUE TITLE
Подбор шрифтов

### ISSUE THREAD

**Issue by f8upd8** [f8upd8/Product503#3]
Нужно выбрать шрифты, описать какой шрифт и куда пойдёт

**Comment by Anatohy** [comment]
https://fonts.google.com/specimen/Orbitron?subset=latin

**Comment by Anatohy** [comment]
https://fonts.google.com/specimen/Orbitron?subset=latin

**Comment by Anatohy** [comment]
https://fonts.google.com/specimen/Electrolize?subset=latin

**Comment by Anatohy** [comment]
https://fonts.google.com/specimen/Turret+Road?subset=latin

**Comment by Anatohy** [comment]
https://fonts.google.com/specimen/Tomorrow?subset=latin

**Comment by Anatohy** [comment]
https://fonts.google.com/specimen/Oxanium?subset=latin

**Comment by Anatohy** [comment]
https://fonts.google.com/specimen/Geo?subset=latin

**Comment by f8upd8** [comment]
Шрифты выше - раскидать по категориям, какие на какую страницу \ для чего.

**Comment by f8upd8** [comment]
Проверил шрифты выше - они все распространяются по свободной лицензии, об этом можете не беспокоится.

**Comment by f8upd8** [comment]
Прошу ещё при выборе перепроверить, есть ли для шрифта кириллица.

**Comment by Anatohy** [comment]
Microsoft YaHei - это шрифт для текста. Остальные еще не выбрал

**Comment by Anatohy** [comment]
https://fontm.com/ronduit-capitals-font/

**Comment by f8upd8** [comment]
> Microsoft YaHei - это шрифт для текста. Остальные еще не выбрал

Для какого текста? Неинформативен.

**Comment by PENIA-2** [comment]
https://fonts.google.com/specimen/Rajdhani?subset=latin

**Comment by PENIA-2** [comment]
https://fonts.google.com/specimen/Roboto

**Comment by PENIA-2** [comment]
https://fonts.google.com/specimen/Mina?subset=latin

**Comment by PENIA-2** [comment]
https://fonts.google.com/specimen/Armata?subset=latin

**Comment by PENIA-2** [comment]
https://fonts.google.com/specimen/Coda?subset=latin

**Comment by Anatohy** [comment]
![fonts](https://user-images.githubusercontent.com/64703855/81008058-78ae5000-8e7c-11ea-873e-ac1c8833fb23.png)

**Comment by Anatohy** [comment]
Какие 2 шрифта вам нравится больше остальных?

**Comment by Anatohy** [comment]
![fonts](https://user-images.githubusercontent.com/64703855/81008932-f1fa7280-8e7d-11ea-9294-bf63376dd588.png)

**Comment by Anatohy** [comment]
Roboto - единственный шрифт с кириллицей. Берём его за стандартный

**Comment by f8upd8** [comment]
Roboto подойдёт для основного текстового контента, но плохо подходит для заголовков, кнопок и надписей.

**Comment by f8upd8** [comment]
Я установил Roboto как стандартный, эффект на шапке можно лицезреть

**Comment by Anatohy** [comment]
![1](https://user-images.githubusercontent.com/64703855/81200431-255c0f00-8fee-11ea-9542-058b4f8497ac.png)

**Comment by Anatohy** [comment]
Шрифт "Tomorrow" - Я выбрал самый тонкий для заголовка. Шрифт удобен тем, что там большое разнообразие в толщине шрифта. Из минусов - только английский язык, но я не вижу в этом большой проблемы. Просто все заголовки будут на английском. И тем более у фонда будет акцент на англоязычных людей

**Comment by f8upd8** [comment]
Установил шрифт "Tomorrow" для всей шапки. Жду оценки.

**Comment by f8upd8** [comment]
Если во всех  местах шрифт устраивает - закрывайте тему.

**Comment by PENIA-2** [comment]
Не согласен. Мне кажется, что шрифт "Tomorrow" слишком угловатый для заголовка и выглядит весьма специфично для сайта такого рода.

В качестве альтернативы я нашёл два шрифта, работающие как в латинице, так и в кириллице:

https://fonts.google.com/specimen/Jura?vfonly&subset=cyrillic
https://fonts.google.com/specimen/Play?subset=cyrillic

**Comment by Anatohy** [comment]
![Jura](https://user-images.githubusercontent.com/64703855/81313128-a4664b80-90b1-11ea-8412-583bc6640842.png)

**Comment by Anatohy** [comment]
Сверху то, что нужно. А снизу Jura, а Play у меня в фотошопе уже есть, я от него отказался еще в 2019 году. Вообщем... я не согласен,  и считаю, что нужно искать дальше

**Comment by Anatohy** [comment]
Ищи тонкие шрифты
